drift-client/src/main/java/com/facebook/drift/client/DriftMethodInvocation.java

```diff
@@ -330,7 +332,11 @@ private synchronized void fail(String reason)
                 invocationAttempts,
                 succinctNanos(ticker.read() - startTime),
                 failedConnections,
-                overloadedRejects);
+                overloadedRejects,
+                attemptedAddresses.stream()
+                        .map(Address::getHostAndPort)
+                        .map(HostAndPort::getHost)
+                        .collect(Collectors.joining(",", "[", "]")));
```

Comment:
Let's store them as `List<Address>` in the `RetriesFailedException` so it can be examined if needed. Also i would recommend simply printing the `List<Address>`, and not only the `HostAndPort`. The `Address` implementation may have additional useful information.